src/rmp/src/zero_slack_strategy.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2025 Google LLC
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include "zero_slack_strategy.h"
+
+#include <iostream>
+
+#include "abc_library_factory.h"
+#include "db_sta/dbNetwork.hh"
+#include "db_sta/dbSta.hh"
+#include "delay_optimization_strategy.h"
+#include "logic_cut.h"
+#include "logic_extractor.h"
+#include "sta/Graph.hh"
+#include "sta/PortDirection.hh"
+#include "sta/Units.hh"
+#include "sta/VerilogWriter.hh"
+#include "unique_name.h"
+#include "utl/Logger.h"
+#include "utl/deleter.h"
+
+namespace rmp {
+
+std::vector<sta::Vertex*> GetNegativeVertices(sta::dbSta* sta)
```

Comment:
GetNegativeEndpoints is more descriptive

Reply:
Done